client/reader/full-post/index.jsx

```diff
@@ -53,6 +53,8 @@ var CommentButton = require( 'components/comment-button' ),
 
 import PostExcerpt from 'components/post-excerpt';
 import { getPostTotalCommentsCount } from 'state/comments/selectors';
+import GalleryView from 'components/tinymce/plugins/wpcom-view/gallery-view';
+import $ from 'jquery/dist/jquery.min.js';
```

Comment:
I think we should avoid pulling in jQuery if at all possible. Will drop follow-up comments on specific usage with suggested alternatives.